src/main/resources/db/changesets/20230307_shedlock.yaml

```diff
@@ -0,0 +1,32 @@
+databaseChangeLog:
+  - changeSet:
+      id: shedlock
+      author: okotsopo
+      changes:
+        - createTable:
+            tableName: shedlock
```

Comment:
huh...interesting that shedlock doesn't handle its own table creation

Reply:
Yeah, a little surprising!  Here's where they instruct you to create it in the docs: https://github.com/lukas-krecan/ShedLock#jdbctemplate